src/main/resources/schemas/ont-metadata.schema.json

```diff
@@ -5,6 +5,9 @@
   "description": "Describes in which form nanopore metadata is received from the lab.",
   "type": "object",
   "properties": {
+    "adapter": {
+      "type": "string"
+    },
```

Comment:
```suggestion
    "adapter": {
      "type": "string",
      "minLength": 1
    },
```

Reply:
it can be empty, as not always an adapter is used. Can make this property not required though...and if it is there, it must not be empty

Reply:
I like the idea of only containing the field if it actually is there.